NotifierTemplateApi/Api/Data/DatabaseTemplateInterface.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Copyright © MageSpecialist - Skeeller srl. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace MSP\NotifierTemplateApi\Api\Data;
+
+use Magento\Framework\Api\ExtensibleDataInterface;
+
+interface DatabaseTemplateInterface extends ExtensibleDataInterface
+{
+    /**
+     * Field template ID
+     */
+    public const ID = 'template_id';
```

Comment:
It is not totally necessary, it can be useful if you need to refer to the field name using a constant instead of a string. Do you think they should be private?

Reply:
As only the DTO implementation uses these constants, I think, it makes sense to move them to DTO class and make private.